src/ta_main.js

```diff
@@ -26,13 +26,26 @@ fetch = function(a, b) {
 };
 
 const routes = [
-    { path: "/user_id/:user_id/term/:term/courses", component: CoursePairings },
-    { path: "/user_id/:user_id/term/:term/Dashboard", component: Dashboard },
+    {
+        path: "/user_id/:user_id/term/:term/courses",
```

Comment:
? Shouldn't these be params?

Reply:
No. We specify the routes this way but when we push the routes, we specify the parameters as a json object and vue router will place it within the route in the correct order. This way, whenever we push a route, we don't need to worry about the order.

Reply:
I'd like to change this. The "route" should be "/Dashboard" and "/courses". Everything else should be a query param.